src/mock_tracer/mock_tracer.ts

```diff
@@ -34,12 +34,15 @@ export class MockTracer extends opentracing.Tracer {
         return span;
     }
 
-    protected _inject(span: MockSpan, format: any, carrier: any): never {
-        throw new Error('NOT YET IMPLEMENTED');
+    protected _inject(span: MockSpan, format: any, carrier: any): void {
+        // throw new Error('NOT YET IMPLEMENTED');
     }
 
-    protected _extract(format: any, carrier: any): never {
-        throw new Error('NOT YET IMPLEMENTED');
+    protected _extract(format: any, carrier: any): opentracing.SpanContext {
+        // throw new Error('NOT YET IMPLEMENTED');
+        let context: opentracing.SpanContext;
+        context = new opentracing.SpanContext ();
+        return context;
```

Comment:
I don't think this is a good change - the implementation is incorrect anyway, so throwing exception is more honest. Otherwise it will just confuse people trying to use this mock tracer.